lib/jorb/backends/file.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Jorb.Backends.File do
+  @moduledoc ~S"""
+  File-based backend for light queueing.
+
+  You probably don't want to use this for production load.
+  """
+  @behaviour Jorb.Backend
+
+  def setup(queue_name) do
+    unless File.exists?(queue_dir(queue_name)) do
+      queue_dir(queue_name) |> File.mkdir_p!()
+    end
+  end
+
+  def enqueue(queue_name, payload) do
+    with now <- :erlang.system_time(:milli_seconds),
+         message_name <- "#{now}.json",
+         {:ok, body} <- Poison.encode(payload),
+         dir <- queue_dir(queue_name),
+         path <- Path.join(dir, message_name),
+         final_payload <- %{receipt_handle: path, body: body},
+         {:ok, encoded} <- Poison.encode(final_payload),
+         :ok <- File.write(path, encoded) do
+      {:ok, final_payload}
+    else
+      err -> err
+    end
+  end
+
+  def dequeue(queue_name) do
```

Comment:
This doesn't actually dequeue items; subsequent calls to `dequeue/1` will return the same results until `finalize/2` is called.  This function is more akin to `first`.

Reply:
`peek` I think is the canonical queue term for "look at the thing on the top"